tracing-futures/Cargo.toml

```diff
@@ -8,10 +8,10 @@ default = ["tokio"]
 
 [dependencies]
 futures = "0.1"
-tokio-trace = "0.1.0"
+tracing = { path = "../tracing" }
```

Comment:
Oh, this is _excellent_! Now depending on `tracing` from git along with these crates will finally work again!

Reply:
yup, however, this is also why there are apparently unrelated changes in this PR --- some of the nursery crates didn't compile with the master version of the core crates (which have breaking changes), so I merged changes in from a branch that updated them to work with the latest.